### PR TITLE
KGPE-D16 test cases description 

### PR DESCRIPTION
**Pull request by sulewskiprzemyslaw**
KGPE-D16:
1. Add coreboot base port test cases description.
2. Add generic test setup file.
3. Add index file.

### CODE REVIEW

File: docs/variants/asus_kgpe_d16/dasharo-compatibility/generic-test-setup.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Dasharo Compatibility: generic test setup
```
Comment (kl3mdeb) [inline docs/variants/asus_kgpe_d16/dasharo-compatibility/generic-test-setup.md:1]: Myślę, że dla lepszej czytelności można napisać tytuł z dużej litery tak jak poniżej

File: docs/variants/asus_kgpe_d16/dasharo-compatibility/generic-test-setup.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+need to execute the setup actions before each independent case.
+
+### Generic test setup
+
+#### Firmware
+
+1. Obtain `FIRMWARE` binary
+    1. you can download it from [releases](../releases.md)
+       section
+    1. or you can build one yourself as shown in the
```
Comment (kl3mdeb) [inline docs/variants/asus_kgpe_d16/dasharo-compatibility/generic-test-setup.md:16]: Lepiej chyba, żeby zdanie nie zaczynało się od or, podpunkty też lepiej wyglądają napisane z dużej litery. Problem też jest z numeracją. Może obie opcje lepiej będzie ująć w jednym zdaniu i punkcie.

File: docs/variants/asus_kgpe_d16/dasharo-compatibility/generic-test-setup.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+       [Building manual](../building-manual.md) section
+1. Flash `FIRMWARE` binary to the DUT
```
Comment (kl3mdeb) [inline docs/variants/asus_kgpe_d16/dasharo-compatibility/generic-test-setup.md:18]: Problem z numeracją.

File: docs/variants/asus_kgpe_d16/dasharo/100-coreboot-base-port.md [hunk 4]
```diff
@@ -0,0 +1,123 @@
...
+**Test steps**
+
+1. Power ON the DUT.
+2. By using the serial console read the booting procedure stage.
+
+**Expected result**
+
+1. The console output should contain string with the phrase:
+
+            bootblock starting (log level: 8)...
```
Comment (kl3mdeb) [inline docs/variants/asus_kgpe_d16/dasharo/100-coreboot-base-port.md:39]: Output konsolowy można ująć w ` lub ```.

File: docs/variants/asus_kgpe_d16/dasharo/100-coreboot-base-port.md [hunk 5]
```diff
@@ -0,0 +1,123 @@
...
+**Test steps**
+
+1. Power ON the DUT.
+2. By using the serial console read the booting procedure stage.
+
+**Expected result**
+
+1. The console output should contain string with the phrase:
+
+            romstage starting (log level: 8)...
```
Comment (kl3mdeb) [inline docs/variants/asus_kgpe_d16/dasharo/100-coreboot-base-port.md:67]: Jak wyżej.

File: docs/variants/asus_kgpe_d16/dasharo/100-coreboot-base-port.md [hunk 6]
```diff
@@ -0,0 +1,123 @@
...
+**Test steps**
+
+1. Power ON the DUT.
+2. By using the serial console read the booting procedure stage.
+
+**Expected result**
+
+1. The console output should contain string with the phrase:
+
+            postcar starting (log level: 8)...
```
Comment (kl3mdeb) [inline docs/variants/asus_kgpe_d16/dasharo/100-coreboot-base-port.md:95]: Jak wyżej.

File: docs/variants/asus_kgpe_d16/dasharo/100-coreboot-base-port.md [hunk 7]
```diff
@@ -0,0 +1,123 @@
...
+**Test steps**
+
+1. Power ON the DUT.
+2. By using the serial console read the booting procedure stage.
+
+**Expected result**
+
+1. The console output should contain string the with phrase:
+
+            ramstage starting (log level: 8)...
```
Comment (kl3mdeb) [inline docs/variants/asus_kgpe_d16/dasharo/100-coreboot-base-port.md:123]: Jak wyżej.

File: docs/variants/asus_kgpe_d16/dasharo-compatibility/generic-test-setup.md (hunk 1 above)
Comment (sulewskiprzemyslaw) [reply docs/variants/asus_kgpe_d16/dasharo-compatibility/generic-test-setup.md:1]: Improved

File: docs/variants/asus_kgpe_d16/dasharo-compatibility/generic-test-setup.md (hunk 2 above)
Comment (sulewskiprzemyslaw) [reply docs/variants/asus_kgpe_d16/dasharo-compatibility/generic-test-setup.md:16]: Changed to subsections

File: docs/variants/asus_kgpe_d16/dasharo-compatibility/generic-test-setup.md (hunk 3 above)
Comment (sulewskiprzemyslaw) [reply docs/variants/asus_kgpe_d16/dasharo-compatibility/generic-test-setup.md:18]: The numbering of the points is done automatically.

File: docs/variants/asus_kgpe_d16/dasharo/100-coreboot-base-port.md (hunk 4 above)
Comment (sulewskiprzemyslaw) [reply docs/variants/asus_kgpe_d16/dasharo/100-coreboot-base-port.md:39]: That type of formatting in test cases description I took from another project. Please compare with novacustom project.

File: docs/variants/asus_kgpe_d16/dasharo/100-coreboot-base-port.md (hunk 5 above)
Comment (sulewskiprzemyslaw) [reply docs/variants/asus_kgpe_d16/dasharo/100-coreboot-base-port.md:67]: That type of formatting in test cases description I took from another project. Please compare with novacustom project.

File: docs/variants/asus_kgpe_d16/dasharo/100-coreboot-base-port.md (hunk 6 above)
Comment (sulewskiprzemyslaw) [reply docs/variants/asus_kgpe_d16/dasharo/100-coreboot-base-port.md:95]: That type of formatting in test cases description I took from another project. Please compare with novacustom project.

File: docs/variants/asus_kgpe_d16/dasharo/100-coreboot-base-port.md (hunk 7 above)
Comment (sulewskiprzemyslaw) [reply docs/variants/asus_kgpe_d16/dasharo/100-coreboot-base-port.md:123]: That type of formatting in test cases description I took from another project. Please compare with novacustom project.

Comment (sulewskiprzemyslaw) [reply docs/variants/asus_kgpe_d16/dasharo/100-coreboot-base-port.md:123]: OK I change that

File: docs/variants/asus_kgpe_d16/dasharo-compatibility/generic-test-setup.md (hunk 3 above)
Comment (miczyg1) [reply docs/variants/asus_kgpe_d16/dasharo-compatibility/generic-test-setup.md:18]: Keep using English in public repos please @kl3mdeb

File: docs/variants/asus_kgpe_d16/dasharo/100-coreboot-base-port.md (hunk 4 above)
Comment (miczyg1) [reply docs/variants/asus_kgpe_d16/dasharo/100-coreboot-base-port.md:39]: THe log level might be different depending on configuration. I wouldn't put it here.

File: docs/variants/asus_kgpe_d16/dasharo/100-coreboot-base-port.md (hunk 5 above)
Comment (miczyg1) [reply docs/variants/asus_kgpe_d16/dasharo/100-coreboot-base-port.md:67]: a/a

File: docs/variants/asus_kgpe_d16/dasharo/100-coreboot-base-port.md (hunk 6 above)
Comment (miczyg1) [reply docs/variants/asus_kgpe_d16/dasharo/100-coreboot-base-port.md:95]: a/a

File: docs/variants/asus_kgpe_d16/dasharo/100-coreboot-base-port.md (hunk 7 above)
Comment (miczyg1) [reply docs/variants/asus_kgpe_d16/dasharo/100-coreboot-base-port.md:123]: a/a

**Review by miczyg1 — CHANGES_REQUESTED** [review]
the test cases do not show which requirements are satisfied by passing the test. I wonder if we should include such descriptions

File: docs/variants/asus_kgpe_d16/dasharo/100-coreboot-base-port.md (hunk 4 above)
Comment (sulewskiprzemyslaw) [reply docs/variants/asus_kgpe_d16/dasharo/100-coreboot-base-port.md:39]: OK, so I limit it to the phrase 'bootblock starting'

File: docs/variants/asus_kgpe_d16/dasharo/100-coreboot-base-port.md (hunk 5 above)
Comment (sulewskiprzemyslaw) [reply docs/variants/asus_kgpe_d16/dasharo/100-coreboot-base-port.md:67]: OK, so I limit it to the phrase 'romstage starting'

File: docs/variants/asus_kgpe_d16/dasharo/100-coreboot-base-port.md (hunk 6 above)
Comment (sulewskiprzemyslaw) [reply docs/variants/asus_kgpe_d16/dasharo/100-coreboot-base-port.md:95]: OK, so I limit it to the phrase 'postcar starting'

File: docs/variants/asus_kgpe_d16/dasharo/100-coreboot-base-port.md (hunk 7 above)
Comment (sulewskiprzemyslaw) [reply docs/variants/asus_kgpe_d16/dasharo/100-coreboot-base-port.md:123]: OK, so I limit it to the phrase 'ramstage starting'

**Comment by sulewskiprzemyslaw** [comment]
> the test cases do not show which requirements are satisfied by passing the test. I wonder if we should include such descriptions

@miczyg1 do You think that we should expand the test case description structure? Or should we expand 'Test description' section?

**Comment by miczyg1** [comment]
> > the test cases do not show which requirements are satisfied by passing the test. I wonder if we should include such descriptions
> 
> @miczyg1 do You think that we should expand the test case description structure? Or should we expand 'Test description' section?

Extending test description would be good

**Comment by sulewskiprzemyslaw** [comment]
> > > the test cases do not show which requirements are satisfied by passing the test. I wonder if we should include such descriptions
> > 
> > 
> > @miczyg1 do You think that we should expand the test case description structure? Or should we expand 'Test description' section?
> 
> Extending test description would be good

Test descriptions have been extended. Is there anything else I should add to them?

**Comment by macpijan** [comment]
@miczyg1 Please approve or request further changes

**Review by miczyg1 — APPROVED** [review]

**Review by kl3mdeb — APPROVED** [review]

**Pull request merged**